src/Orleans/Properties/AssemblyInfo.cs

```diff
@@ -44,3 +44,4 @@
 [assembly: InternalsVisibleTo("Tester.SQLUtils")]
 [assembly: InternalsVisibleTo("Tester.ZooKeeperUtils")]
 [assembly: InternalsVisibleTo("AWSUtils.Tests")]
+[assembly: InternalsVisibleTo("ServiceBus.Tests")]
```

Comment:
That's needed because I need to init the serialization env for the tests, specifically to BufferPool.InitGlobalBufferPool(new MessagingConfiguration(false));  , which is used by dataAdapter to serialize queue message into cache message body bits. And BufferPool.InitGlobalBufferPool(new MessagingConfiguration(false)); is an internal method.